packages/cli/README.md

```diff
@@ -160,12 +160,17 @@ s?: TRACE_EVENT_SCOPE;
 # Commands
 
 <!-- commands -->
-* [`tracerbench compare`](#tracerbench-compare)
-* [`tracerbench create-archive`](#tracerbench-create-archive)
-* [`tracerbench help [COMMAND]`](#tracerbench-help-command)
-* [`tracerbench marker-timings`](#tracerbench-marker-timings)
-* [`tracerbench report`](#tracerbench-report)
-* [`tracerbench trace`](#tracerbench-trace)
+- [Usage](#Usage)
```

Comment:
The prepack task did this on the push hook.  Not sure why this happened. @lynchbomb did you manually edit something that was autogenerated or did a config get messed up with this change?